### ISSUE TITLE
Is aarch64 fuzzing supported?

### ISSUE THREAD

**Issue by alexcrichton** [google/oss-fuzz#11463]
Hello! A colleague of mine on Wasmtime [asked the same question](https://github.com/google/oss-fuzz/issues/3854) a few years ago and I've been checking in every now and then to check on the status. I see now there's two project configured in their `project.yaml` for aarch64 ([lua](https://github.com/google/oss-fuzz/blob/6c19a0f229c244d41562ece92f9baac6b72426f9/projects/lua/project.yaml#L10) and [tinyusb](https://github.com/google/oss-fuzz/blob/6c19a0f229c244d41562ece92f9baac6b72426f9/projects/tinyusb/project.yaml#L16)), although lua's build seems to be failing on aarch64 while tinyusb's is succeeding. I also see that the [aarch64 issue](https://github.com/google/oss-fuzz/issues/8164) got a lot of cc's last year but activity has died down a bit.

In any case I wanted to ask the question (feel free to close this with a simple "not yet") to see if the status has changed in the meantime. Additionally if you're looking for test subjects Wasmtime would definitely be willing to be a guinea pig here too.

**Comment by jiliguluss** [comment]
I recently encountered the same issue, and I would like to inquire about running an arm32 or aarch64 libfuzzer target on QEMU.
Someone shared how to obtain aarch64 libfuzzer target through cross-compiling on Linux, but I did not find instructions for running libfuzzer target on qemu.
I noticed in previous issues that there was mention of compatibility problems between ASan and QEMU.
If there is any progress on supporting arm32 or aarch64, that would be great.
